cowprotocol/cowswap
# fix: fix connect wallet modal while changing wallet

# Summary

Click by "Change wallet" opens a modal behind Account modal

![image](https://github.com/cowprotocol/cowswap/assets/7122625/6d2feed8-9919-4721-988d-c3e10bdd1712)

  # To Test

1. Connect wallet
2. Open account modal
3. Click "Change wallet"
- [ ] Wallet selection modal should be displayed on top of Account modal
